src/app/onion/dashboard/dashboard.component.ts

```diff
@@ -48,4 +87,41 @@ export class DashboardComponent {
 
     this.router.navigateByUrl(url);
   }
+
+  /**
+   * Retrieves an array of learningObjects to populate the draft and released list of learning objects
+   * @param filters
+   * @param query
+   */
+  async getLearningObjects(filters?: any, text?: any): Promise<LearningObject[]> {
+    this.loading = true;
+    return this.learningObjectService
+    .getLearningObjects(this.auth.username, filters, text)
+    .then((children: LearningObject[]) => {
+      this.loading = false;
+      return children;
+    });
+  }
+
+  /**
+   * Retrieves an array of learningObjects to populate the draft list of learning objects
+   * This will only retrieve the drafts and will not retrieve any revisions of a learning object
+   * @param filters
+   * @param text
+   */
+  async getDraftLearningObjects(filters?: any, text?: any): Promise<LearningObject[]> {
+    this.loading = true;
+    return this.learningObjectService
+    .getDraftLearningObjects(this.auth.username, filters, text)
+    .then((children: LearningObject[]) => {
+      this.loading = false;
+      return children;
+    });
+  }
+
+  async performSearch(text: string) {
+    this.releasedLearningObjects = await this.getLearningObjects({status: ['released']}, text);
+    this.workingLearningObjects = await this.getDraftLearningObjects(
+      {status: ['unreleased', 'proofing', 'review', 'rejected', 'waiting']}, text);
```

Comment:
What was the reasoning behind filtering draft Learning Objects by all statuses instead of providing no filters if none are specified by the user?

By default, when the `draftsOnly` filter is specified, that included Learning Objects in all of the statuses listed.